php_phongo.c

```diff
@@ -1561,13 +1561,13 @@ static void php_phongo_cursor_iterator_dtor(zend_object_iterator *iter TSRMLS_DC
 {
 	php_phongo_cursor_iterator *cursor_it = (php_phongo_cursor_iterator *)iter;
 
+	php_phongo_cursor_free_current(cursor_it->cursor);
+
```

Comment:
Since this only frees `cursor->visitor_data.zchild` if it's not null, was there a specific reason to move it up? I've no objection if it was just for style points, but I'm curious if there was another reason.